client/components/Links.css

```diff
@@ -0,0 +1,9 @@
+/*li {
+  list-style-type: none;
+  border: 1px solid black;
+}
+
+span.score {
+    float: right;
+}
+*/
```

Comment:
To make tests work with react components that import non-JS stuff (via webpack), one strat is to break component up into two files:

* Simple component: JS(X) only, no importing CSS
* Complex component: uses simple component, connects it to Redux, imports CSS, etc.